arangod/Cluster/CreateCollection.cpp

```diff
@@ -101,6 +101,8 @@ CreateCollection::~CreateCollection() {};
 
 bool CreateCollection::first() {
 
+  ActionBase::first();
```

Comment:
This is bad.  Implies burden on programmer to remember this call in every derived object.  And current implementation is redundant to MaintenanceWorker::nextState().